xapi-profiles-communication.md

```diff
@@ -176,6 +176,12 @@ function described in pseudocode below with the Statement and all the Statement
 from the Profile. This function returns an outcome and an array of Statement Templates. To
 interpret the results, consult the table after the algorithm definition.
 
+Comparison between `values` parsed from the Statement and the `any`, `all` and `none` values
+MUST be based on the data comparison requirements defined within the profile's xAPI Version.
+* xAPI data comparison is based off of the optional `xAPIVersion` Profile value.
```

Comment:
Maybe I'm a nitpicking but for me sounds awkward *based off of* instead of *based on*.

Reply:
agreed, rewording incoming